src/test/java/org/earthtime/UPb_Redux/utilities/BrowserControlTest.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2015 CIRDLES.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.earthtime.UPb_Redux.utilities;
+
+import org.junit.Test;
+
+/**
+ *
+ * @author parizotclement
+ */
+public class BrowserControlTest {
+    
+    /**
+     * Integration Test of class BrowserControl
+     * Testing the browser of a Windows or Mac platforms
+     */
+    @Test
+    public void testException() {
+        
+        BrowserControl.displayURL("http://www.javaworld.com");
```

Comment:
You can't open a browser from an automated test unless you have a way to close it. This needs to be refactored or removed.